clang/test/CodeGen/aarch64-sve2p1-intrinsics/acle_sve2p1_psel.c

```diff
@@ -5,6 +5,11 @@
 // RUN: %clang_cc1 -fclang-abi-compat=latest -triple aarch64-none-linux-gnu \
 // RUN:   -target-feature +sve2p1 -S -O1 -Werror -emit-llvm -o - -x c++ %s | FileCheck %s -check-prefix=CPP-CHECK
 // RUN: %clang_cc1 -fclang-abi-compat=latest -triple aarch64-none-linux-gnu -target-feature +sve2p1 -S -disable-O0-optnone -Werror -Wall -o /dev/null %s
+// RUN: %clang_cc1 -fclang-abi-compat=latest -triple aarch64-none-linux-gnu \
```

Comment:
I would imagine these tests also need to be updated to use arm_streaming and the 
#ifdef IGNORE_STREAMING_ATTR
#define __attribute__(...)
#endif

Reply:
As per https://github.com/ARM-software/acle/pull/217/files#diff-516526d4a18101dc85300bc2033d0f86dc46c505b7510a7694baabea851aedfaR11352-R11358
should this be `__arm_streaming_compatible` instead of `__arm_streaming` for psel?